modules/core/src/test/java/org/apache/ignite/internal/IgniteClientRejoinTest.java

```diff
@@ -354,12 +354,12 @@ public void testClientsReconnectDisabled() throws Exception {
      */
     private class DiscoverySpi extends TcpDiscoverySpi {
         /** {@inheritDoc} */
-        @Override protected void writeToSocket(Socket sock, TcpDiscoveryAbstractMessage msg, byte[] data,
-            long timeout) throws IOException {
+        @Override protected void writeToSocket(Socket sock, byte[] data,
```

Comment:
@aleksey-kurinov , should it get on-lined?

Reply:
fixed

Reply:
fixed